shared/src/common/types/user/user.ts

```diff
@@ -0,0 +1,24 @@
+enum Role {
```

Comment:
I think for enum we should create a different file. Also, I prefer to specify with prefix an enum like this if we use it only for the User interface
Example.: UserRole

Reply:
Great suggestion!

Reply:
Good point, thanks! I've redefined the types.